modules/controllers/snapshots-controller.ts

```diff
@@ -77,18 +76,30 @@ export function SnapshotsController(tracer?: any) {
                 throw new Error(`Chain not configured: ${chain}`);
             }
 
-            const vaultSubgraphClient = getVaultSubgraphClient(balancerV3);
-            const entries = await syncSnapshotsV3(vaultSubgraphClient, chain);
+            const vaultSubgraphClient = getVaultSubgraphClient(balancerV3, chain);
+            const entries = await syncSnapshots(vaultSubgraphClient, 'SNAPSHOTS_V3', chain);
```

Comment:
shouldnt this call syncLatest: true?

Reply:
i'm setting "safe defaults" here to always sync latest only when no options are given